### ISSUE TITLE
__indexOf and coverage report

### ISSUE THREAD

**Issue by genu** [jashkenas/coffeescript#4268]
Related to #2562 

``` javascript
var __indexOf = [].indexOf || function(item) { for (var i = 0, l = this.length; i < l; i++) { if (i in this && this[i] === item) return i; } return -1; };
```

I realize that this is a workaround for lack of `indexOf` in IE < 9.0, however, coverage reports are affected since the `.indexOf()` is defined in most browsers.

Is there a way to disable IE less than 9 altogether to disable the extra indexOf definition?

**Comment by carlsmith** [comment]
I don't think there is a way currently, but this is one of the things that probably should be optional.

**Comment by genu** [comment]
Perhaps, a workaround would be useful to avoid having incomplete coverage.

**Comment by ryoqun** [comment]
@genu As a another approach, you can try coffee-coverage. :)

**Comment by jashkenas** [comment]
Is it time to drop support for IE < 9?

**Comment by GeoffreyBooth** [comment]
Fixed in #4526.
